scrapper.py

```diff
@@ -41,21 +44,38 @@ def get_search_urls(self):
         """
         Returns seed_urls param
         """
-        pass
+        return self.seed_urls
 
 
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
```

Comment:
this will work
however, there is a simpler way to implement it without using try/except